src/scss/components/navigation/navigation.scss

```diff
@@ -32,24 +34,28 @@
     position: relative;
     margin-right: 60px;
 
-    &::after {
-      content: '';
-      position: absolute;
-      top: calc(50% - 7px);
-      right: -18px;
-      width: 7px;
-      height: 7px;
-      background: transparent;
-      border-bottom: 2px solid $brand-color;
-      border-right: 2px solid $brand-color;
-      transform: rotate(45deg);
-      transition: border-color 0.3s ease, transform 0.3s ease, top 0.3s ease;
-      cursor: pointer;
+    & .sub-menu-indicator {
+      position: relative;
+
+      &::after {
+        content: '';
+        position: absolute;
+        top: calc(50% - 7px);
+        right: -18px;
+        width: 7px;
+        height: 7px;
+        background: transparent;
```

Comment:
Recommended to use non-words color definitions (e.g. rgba). In case there will be a mixin with function to lighten or darken a color there could be problems. Please note that lighten and darken functions itself work fine, but in some cases, they could fail.